modules/packages/HDFS.chpl

```diff
@@ -265,6 +265,17 @@ module HDFS {
       return oldValue - 1;
     }
 
+    deprecated "open with a style argument is deprecated, please rely on the default value for the argument (a new strategy is being developed)"
```

Comment:
I don't think so.  The last resort marks were due to an earlier state of the branch that has since been fixed by not using a default value with the style arguments.  I think the right call is to remove them now.